buildah.go

```diff
@@ -28,7 +28,7 @@ const (
 	Package = "buildah"
 	// Version for the Package.  Bump version in contrib/rpm/buildah.spec
 	// too.
-	Version = "1.19.0"
+	Version = "1.19.2"
```

Comment:
I'm concerned that the version wasn't originally 1.19.1 here and in bulidah.spec.  I'm not sure how that happened, but the end result looks OK....